pywinauto/controls/HwndWrapper.py

```diff
@@ -1223,6 +1223,76 @@ def scroll(self, direction, amount, count = 1, retry_interval = None):
     # Non PEP-8 alias
     Scroll = scroll
 
+   #-----------------------------------------------------------
+    def get_scroll_info(self, hwnd, fnBar, lpsi):
+        win32functions.GetScrollInfo(hwnd, fnBar, lpsi)
```

Comment:
It looks like just a wrapper around `GetScrollInfo`. `HwndWrapper` method should NOT have `hwnd` or any kind of such parameters because its interface is high level. `GetScrollInfo` is low-level function. So you must use `self.handle` instead of external (which?) hwnd. And so on.

`SCROLLINFO` structure (`lpsi` param) fields would also be interpreted in a more human readable form.